SplunkCloud/terraform-provider-dyn
# CWORKS-1119 - Update to use latest go-dynect code (#5)

* make: Add website + website-test targets

* add checks for record ID

* Add creds for meta user + .gitigore

* added ci steps and dockerfile

* Using the latest version of the go-dynect lib

* check before delete

* Check if the ID exists before looking for the ID

* Trying ANYRecord rather than ALLRecord

* Updating version of go-dynect
